data-processing/pdf/process_pdf.py

```diff
@@ -0,0 +1,236 @@
+import requests
+import logging
+from typing import List, Dict
+import sys
+from tempfile import TemporaryDirectory
+import subprocess
+import time
+
+import pdf.grobid_client
+from common.models import BoundingBox as BoundingBoxModel
+from common.models import init_database_connections
+from common.types import CitationData, CitationLocation, SerializableReference, BoundingBox, Matches, Match, Symbol
+from entities.symbols.types import SymbolData, SymbolWithId, SymbolId
+from entities.citations.utils import upload_citations, extract_ngrams, ngram_sim
+from entities.symbols.utils import upload_symbols
+
+class PdfStructureParser(object):
```

Comment:
nit pick: afaik, `(object)` object base shouldn't be necessary for a Python 3 runtime?